packages/cli/src/__tests__/main.test.tsx

```diff
@@ -0,0 +1,111 @@
+jest.mock('enquirer', () => ({
```

Comment:
nit: defining these here it will still get hoisted below imports wont it? might as well define below them?